server/odc-server/src/main/resources/data.sql

```diff
@@ -732,6 +732,7 @@ INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES
 INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.job-heart-timeout-seconds', '300', 'job heart timeout seconds, job will failed or retrying' ) ON DUPLICATE KEY UPDATE `id` = `id`;
 INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.executor-waiting-to-run-threshold-seconds', '3', 'time threshold of executor waiting to run, for controller schedule rate' ) ON DUPLICATE KEY UPDATE `id` = `id`;
 INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.executor-waiting-to-run-threshold-count', '10', 'amount threshold of executor waiting to run, for controller schedule rate' ) ON DUPLICATE KEY UPDATE `id` = `id`;
+INSERT INTO config_system_configuration ( `key`, `value`, `description` ) VALUES( 'odc.task-framework.limit-running-job-total-memory-in-milli-bytes', '0', 'limit running job total memory in milli bytes, for controller schedule rate in process model' ) ON DUPLICATE KEY UPDATE `id` = `id`;
```

Comment:
1. milli means 1/1000

2. naming should be a noun,  limit--blabla  looks like do some action

Reply:
okey, change to  `odc.task-framework.running-job-total-memory-threshold-in-mb`